src/gui/generalsettings.ui

```diff
@@ -279,8 +363,6 @@
   <tabstop>newExternalStorage</tabstop>
   <tabstop>showInExplorerNavigationPaneCheckBox</tabstop>
   <tabstop>crashreporterCheckBox</tabstop>
-  <tabstop>updateChannel</tabstop>
-  <tabstop>restartButton</tabstop>
```

Comment:
i have no idea why this is like that. When i click to edit the tabsstops in designer they are shown?